### PR TITLE
Adding new APIs

### PR DESCRIPTION
**Pull request by TechPunk316**
# What

Added the following sections and APIs:

- Created a 'DVLA APIs' section 
- Added 'Vehicle Enquiry Service' and 'Authentication' APIs to DVLA APIs
- Added London Borough of Hackney 'Addresses API' to Local government section
- Added 'Bank Holidays API' to GDS section

# Why

Adding the latest APIs that have been submitted

# How to review

Run locally to check that formatting and links are working

### CODE REVIEW

File: source/central/dvla/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: DVLA APIs
+weight: 50
+---
+
+# DVLA APIs
+
+The API catalogue contains the following DVLA APIs:
+
+- [DVLA Vehicle Enquiry Service](Link TBC)
+- [DVLA Authentication](Link TBC]
```
Comment (arnau) [inline source/central/dvla/index.html.md.erb]: What do you need to know for these two links?

File: source/local/London-Borough-Hackney/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: West Berkshire Council API
+weight: 101
+---
+
+# London Borough of Hackney: Addresses
+
+**Base UR**
```
Comment (arnau) [inline source/local/London-Borough-Hackney/index.html.md.erb]: Typo: Should say "Base URL"

Comment (TechPunk316) [reply source/local/London-Borough-Hackney/index.html.md.erb]: Done :)

File: source/central/dvla/index.html.md.erb (hunk 1 above)
Comment (TechPunk316) [reply source/central/dvla/index.html.md.erb]: I'm having trouble creating the proper file structure for the DVLA pages. This is something we can chat about and then we can add the links.

**Pull request merged**